decidim-admin/app/commands/decidim/admin/process_participatory_space_private_user_import_csv.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Decidim
+  module Admin
+    class ProcessParticipatorySpacePrivateUserImportCsv < Rectify::Command
+      # Public: Initializes the command.
+      #
+      # form - the form object containing the uploaded file
+      # current_user - the user performing the action
+      # private_users_to - The private_users_to that will hold the user role
+      def initialize(form, current_user, private_users_to)
+        @form = form
+        @current_user = current_user
+        @private_users_to = private_users_to
+      end
+
+      # Executes the command. Broadcasts these events:
+      #
+      # - :ok when everything is valid.
+      # - :invalid if the form wasn't valid and we couldn't proceed.
+      #
+      # Returns nothing.
+      def call
+        return broadcast(:invalid) unless @form.valid?
+
+        process_csv
+        broadcast(:ok)
+      end
+
+      private
+
+      def process_csv
+        CSV.foreach(@form.file.path) do |row|
```

Comment:
I think you can do something like this:

```suggestion
        CSV.foreach(@form.file.path) do |email, user_name|
```

And avoid the next two lines